addons/udes_advanced_packaging/__manifest__.py

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "UDES Advanced Packaging",
+    "summary": "udes stock",
+    "description": "Advanced packaging features for UDES- Odoo 14",
```

Comment:
```suggestion
    "description": "Advanced packaging features for UDES - Odoo 14",
```

Reply:
:laughing:

Reply:
Fixed, albeit without committing the suggestion

Reply:
yeah i never like using the commit suggestion thing, it just makes it easier to see what changed :+1:

Reply:
:100: